tools/mo/requirements_caffe.txt

```diff
@@ -1,6 +1,6 @@
 networkx~=2.5; python_version <= "3.6"
 networkx<=2.8.8; python_version > "3.6"
-numpy>=1.16.6,<=1.24.0
+numpy>=1.16.6,<1.25.0
```

Comment:
can / should we remove direct dependency here and rely on OpenVINO runtime?
Or at least replace this line with:
```
-r ../../src/bindings/python/requirements.txt
```
?

Reply:
No, we should not remove this file or change this file. These versions correspond to validated versions of dependencies for `openvino-dev` and to have `<` is incorrect because there is no exact version.

Reply:
I mean you exactly the same dumpy version in multiple files - why do we need to duplicate them?

Reply:
In this case we have changed order of installed dependencies. I mean numpy will be installed first, and after that fw using numpy is installed. It can affect us.
Better to have standalone list.

Reply:
We can create the common list of reqs files for all openvino-dev tools. Now we even have duplicates among different tools.
That can be the first step how we can remove duplicates.

Reply:
I proposed to use https://waylonwalker.com/python-nested-requires/

Reply:
May be it is a good thing. We can think of it.